src/app/app.rx.ts

```diff
@@ -0,0 +1,5 @@
+export { Observable } from 'rxjs/Observable';
+export { Subscription } from 'rxjs/Subscription';
+
+import 'rxjs/add/operator/map';
+import 'rxjs/add/observable/timer';
```

Comment:
Because if you're going to use Observables anywhere else in your app, it'll be a PITA to maintain. Also it abstracts away where you're getting your Observable from, sorta.

I used to take the approach you're proposing, but it ended up being a nightmare.